cmd/convox/env_ut_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestParseEnvLine(t *testing.T) {
```

Comment:
nit: This test can go with the other env tests inside `env_test.go`. Nice to keep them together in one file.